Dockerfile

```
@@ -1,2 +1,5 @@
 FROM ubuntu:20.04
-RUN apt-get update && apt-get install --yes --no-install-recommends cowsay
\ No newline at end of file
+RUN apt-get update && \
+    apt-get install --yes --no-install-recommends cowsay=3.04 && \
```

This version does not exists (yet) in ubuntu: https://packages.ubuntu.com/search?suite=focal&searchon=names&keywords=cowsay

```suggestion
    apt-get install --yes --no-install-recommends cowsay=3.03 && \
```